app/controllers/partners_controller.rb

```diff
@@ -73,6 +74,13 @@ def invite
     redirect_to partners_path, notice: "#{partner.name} invited!"
   end
 
+  def recertify_partner
+    @partner = current_organization.partners.find(params[:id])
+    @partner.recertification_required!
+    DiaperPartnerClient.put(@partner.attributes)
```

Comment:
We should be careful with calls like these - if the network request fails the user would never know.

Reply:
yes. good point. we could move this to a background task. We have an `error` state now that we could use if the connection times out or doesn't return 200 OK.

Reply:
we could ensure that DiaperPartnerClient returns a response that we can use to check/switch on the response code. moving this to a background job could be another possibility.  @mdworken are you requesting a change or providing a word of caution?

Reply:
@chaserx Since it shouldn't be too hard to add, let's save the response, check that it's a 200, and error if it isn't.

Reply:
sorry it's been so long. @mdworken I think I've addressed your concerns here.